index.js

```diff
@@ -39,20 +38,22 @@ Interface.createInterface = function( opt ) {
   var rl = readline.createInterface( filteredOpt );
 
   // Fix bug with refreshLine
-  rl._refreshLine = _.wrap(rl._refreshLine, function( func ) {
-    func.call(rl);
+  var _refreshLine = rl._refreshLine;
+  rl._refreshLine = function() {
+    _refreshLine.call(rl);
+
     var line = this._prompt + this.line;
     var cursorPos = this._getCursorPos();
 
     readline.moveCursor(this.output, -line.length, 0);
     readline.moveCursor(this.output, cursorPos.cols, 0);
-  });
+  };
 
   // Returns current cursor's position and line
   rl._getCursorPos = function() {
     var columns = this.columns;
     var strBeforeCursor = this._prompt + this.line.substring(0, this.cursor);
-    var dispPos = this._getDisplayPos(chalk.stripColor(strBeforeCursor));
+    var dispPos = this._getDisplayPos(strBeforeCursor);
```

Comment:
Remove moot stripping as it's being stripped first thing inside the method anyways.